README.md

```diff
@@ -59,19 +59,6 @@ npm run server:dev:hmr
 ```
 go to [http://0.0.0.0:3000](http://0.0.0.0:3000) or [http://localhost:3000](http://localhost:3000) in your browser
 
-# Nitrous Quickstart
```

Comment:
I added nitrous back to the end of `README.md` and left a section for other quick start guides